web/src/search.ts

```diff
@@ -12,15 +12,18 @@ import * as search_suggestion from "./search_suggestion";
 // Exported for unit testing
 export let is_using_input_method = false;
 
-export function set_search_bar_text(text) {
+export function set_search_bar_text(text: string): void {
     $("#search_query").val(text);
 }
 
-function get_search_bar_text() {
-    return $("#search_query").val();
+function get_search_bar_text(): string {
+    return $("#search_query").val() as string;
```

Comment:
Refer to https://github.com/zulip/zulip/pull/29354 to know the right way to convince TypeScript and ensure better runtime safety.